trove/render/simple_csv.py

```diff
@@ -83,6 +83,8 @@ def _column_paths(self) -> Iterator[Propertypath]:
                 _expected_card_types = set(self.trove_params.cardsearch_type_iris())
             else:
                 _expected_card_types = set()
+            if '*' in self.trove_params.attrpaths_by_type:
+                _pathlists.append(self.trove_params.attrpaths_by_type['*'])
```

Comment:
readability nit: this is inserted between where `_expected_card_types` is set and used, despite having nothing to do with that -- would flow better inserted at the start of the `if self.trove_params is not None:` block, leaving the more complicated type-specific logic uninterrupted